hooks/pre-command

```diff
@@ -14,6 +14,8 @@ ecr_exists() {
 }
 
 upsert_ecr() {
+  printenv
```

Comment:
We should remove this, as sensitive information can be stored in environment variables

Reply:
My Bad, leftover from testing.... Removed.
Thanks for picking this.